### PR TITLE
RSS: Add config reference links

### PR DESCRIPTION
**Pull request by bholmesdev**
#### Description (required)

Add links to the `@astrojs/rss` README for a complete configuration reference.

- Link from the step-by-step guide to call out all config options
- Link from the `items` rendering guide to call out the items reference

#### Related issues & labels (optional)

- Related: https://github.com/withastro/astro/pull/12157

### CODE REVIEW

**Comment by bholmesdev** [comment]
All very clear! Just removing `customData` since it's rarely used and best left to config reference

File: src/content/docs/en/guides/rss.mdx [hunk 1]
```diff
@@ -77,7 +80,8 @@ The RSS feed standard format includes the following values for each published it
 - `link`: A URL to the original source of the entry. (optional)
 - `pubDate`: The date of publication of the entry. (optional)
 - `content`: The full content of your post. (optional)
-- `customData`: A field for including any extra data, such as other frontmatter properties from your blog posts.
+
+<ReadMore>See [`item` configuration reference](https://github.com/withastro/astro/tree/main/packages/astro-rss#items) for a complete list of options.</ReadMore>
```
Comment (yanthomasdev) [inline src/content/docs/en/guides/rss.mdx:84]: I believe the option is actually `items`? I am not sure if you meant the configuration reference for an *individual* item, if so, then it should not contain backticks. Also, nit: missing "the".
```suggestion
<ReadMore>See the [`items` configuration reference](https://github.com/withastro/astro/tree/main/packages/astro-rss#items) for a complete list of options.</ReadMore>
```

File: src/content/docs/en/guides/rss.mdx [hunk 2]
```diff
@@ -66,9 +67,11 @@ The package [`@astrojs/rss`](https://github.com/withastro/astro/tree/main/packag
...
 </Steps>
 
+<ReadMore>See the [`@astrojs/rss` README](https://github.com/withastro/astro/tree/main/packages/astro-rss) for the full configuration reference.</ReadMore>
+
 ## Generating `items`
 
-The `items` field accepts a list of RSS feed objects, which can be generated from content collections entries using `getCollection()` or from your page files using `pagesGlobToRssItems()`. 
+The `items` field accepts a list of RSS feed objects, which can be generated from content collections entries using `getCollection()` or from your page files using `pagesGlobToRssItems()`.
 
 The RSS feed standard format includes the following values for each published item:
```
Comment (sarah11918) [inline src/content/docs/en/guides/rss.mdx:76]:
```suggestion
The RSS feed standard format includes metadata for each published item, including values such as:
```
So, my main concern is even with the link below, this sounds like an exhaustive, complete list. Anything to soften that up a bit here would be fine!

File: src/content/docs/en/guides/rss.mdx [hunk 3]
```diff
@@ -66,9 +67,11 @@ The package [`@astrojs/rss`](https://github.com/withastro/astro/tree/main/packag
     ```
 </Steps>
 
+<ReadMore>See the [`@astrojs/rss` README](https://github.com/withastro/astro/tree/main/packages/astro-rss) for the full configuration reference.</ReadMore>
```
Comment (sarah11918) [inline src/content/docs/en/guides/rss.mdx:70]: Now I **really** want this turned into an integration, so we just have the standard integration guide page in docs!!

File: src/content/docs/en/guides/rss.mdx (hunk 1 above)
Comment (sarah11918) [reply src/content/docs/en/guides/rss.mdx:84]: I believe Yan is correct here, it sounds like the intention is to point to the reference property!

File: src/content/docs/en/guides/rss.mdx (hunk 3 above)
Comment (bholmesdev) [reply src/content/docs/en/guides/rss.mdx:70]: Yup, totally agree!

**Review by sarah11918 — APPROVED** [review]
LGTM! Thanks for the update Ben!

**Pull request merged**